deploy/hardware/generic.xml

```diff
@@ -11,10 +11,7 @@
     <apic/>
     <pae/>
   </features>
-  <cpu mode='custom' match='exact'>
-    <model fallback='allow'>kvm64</model>
-    <feature policy='require' name='vmx'/>
-  </cpu>
+  <cpu mode='host-passthrough'/>
```

Comment:
Maybe we could add an optional switch to the python file. So that it by default uses intel, and otherwise amd parameters.